lib/Elevate/Blockers/Python.pm

```diff
@@ -0,0 +1,30 @@
+package Elevate::Blockers::Python;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Blockers::Python
+
+Blocker to check if Conflicting python interpreters are installed.
+
+=cut
+
+use cPstrict;
+
+use parent qw{Elevate::Blockers::Base};
+
+use Cpanel::Pkgr  ();
+
+sub check ($self) {
+    my $pkg = Cpanel::Pkgr::what_provides('python36');
+    return unless $pkg && Cpanel::Pkgr::is_installed($pkg);
+    return $self->has_blocker( <<~"END" );
+    python36 packages have been detected as installed.
```

Comment:
not sure why we use plural here for `packages` as we only detected `one`?

Reply:
but yeap if `python36` is installed, very likely you have some other `python36-*` packages installed too :-)

Reply:
Will push commit to try to address it